guide/web-animation.md

```diff
@@ -0,0 +1,54 @@
+## Web Animation
+
+### Description
+
+Webots can record a simulation to an interactive 3D `HTML` page.
+The result is similar to a movie with playback controls, except that
+you can change the viewpoint at any time.
+
+The mechanism is similar to the [web scene export](web-scene.md),
+except that a supplementary `JSON` animation file containing the moving objects positions is exported.
+
+The [figure bellow](#screenshot-of-a-web-animation-page-generated-by-webots)
+shows an example of an animation page generated by Webots.
+
+
+%figure "Screenshot of a Web animation page generated by Webots"
+
+![screenshot-web-animation.png](images/screenshot-web-animation.png)
+
+%end
+
+
+### How to export a Web Animation
+
+Select the `File / Make HTML5 Animation...` menu item and choose the target `HTML` file.
+Webots is asking to playback the resulting file in the default browser (from the OS settings).
+
+**Note**:
+The `X3D` file, the `JSON` animation file and the required textures are exported in the same directory as the target `HTML` file.
+
+**Note**:
+The playback option may not work correctly depending on your default Web browser
+(cf. [section bellow](#remarks-on-the-used-technologies-and-their-limitations))
+
+**Note**:
+A [Supervisor](supervisor-programming.md) can start/stop an animation from as well.
+
+
+### How to embed a Web Animation in your Website
+
+Please refer to [this section](web-scene.md#how-to-embed-a-web-scene-in-your-website).
+
+
+### Limitations
+
+Webots is exporting only the `Transform.translation`, `Transform.rotation` and `LED.color` fields.
+Modifying the other VRML fields, adding/removing nodes are not recorded in the animation file.
```

Comment:
I would not use the '/' in the documentation.
Instead of the verb I would use the noun: modification, insertion, deletion.
Otherwise this sentence sounds a little bit strange.